dask/bag/tests/test_bag.py

```diff
@@ -300,6 +306,42 @@ def test_map_partitions():
     assert b.map_partitions(lambda a: len(a) + 1).name != b.map_partitions(len).name
 
 
+def test_map_partitions_with_kwargs():
+    corpus = '''
```

Comment:
I rather like this test case since it's much closer to my original use case, and not a contrived number normalizer; however let me know if it's too weird to keep...